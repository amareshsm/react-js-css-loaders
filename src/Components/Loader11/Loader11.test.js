import React from 'react'
import renderer from 'react-test-renderer'
import { Loader11 } from '.'
it('renders correctly', () => {
  const tree = renderer.create(<Loader11 />).toJSON()
  expect(tree).toMatchSnapshot()
})
