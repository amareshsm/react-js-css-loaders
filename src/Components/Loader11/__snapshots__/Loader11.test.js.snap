// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <style>
    [object Object]
  </style>,
  <div
    className="loader-div eleven"
  >
    <div
      className="loader11"
    />
  </div>,
]
`;
